massa-bootstrap/src/server.rs

```diff
@@ -187,14 +188,38 @@ impl BootstrapServer {
                 }
 
                 // listener
-                res_connection = listener.accept(&whitelist, &blacklist) => {
-                    let (dplx, remote_addr) = if res_connection.is_ok() {
-                        res_connection.unwrap()
+                res_connection = listener.accept() => {
+                        let (mut server, remote_addr) = if res_connection.is_ok() {
```

Comment:
Indentation seems wrong here, maybe you didn't run fmt but its probably the formatter bugging out because of the select or the test scopes. If that's the case you have to do it manually.

Reply:
Otherwise lgtm, will merge when this is fixed 👍

Reply:
(And CI is passing)

Reply:
done